scamp/serviceproxy.go

```diff
@@ -24,7 +24,7 @@ type ServiceProxyDiscoveryExtension struct {
 	Vmin   int           `json:"vmin"`
 	Vmaj   int           `json:"vmaj"`
 	AcSec  []interface{} `json:"acsec"`
-	AcName []string      `json:"acname"`
+	AcName []interface{} `json:"acname"`
```

Comment:
This should be fine for now, but I'd rather get clarity regarding what the expected format is for this in scamp and use the correct type(s) rather than an empty interface